src/tfpython.c

```diff
@@ -466,7 +466,7 @@ struct Value *handle_python_command( String *args, int offset )
 	return pyvar_to_tfvar( pRc );
 }
 
-struct Value *handle_python_function( conString *args )
```

Comment:
I really dont understand why we need that empty argument..

Whats the error without it?

Reply:
Looking back, it doesn't appear to have been needed. I think I tried to "fix" a compile error by changing something in expr.c first, when I only needed to address the one below in tfpython.c.

When compiling with GCC-14, I got this error:

tfpython.c: In function ‘pyvar_to_tfvar’:
tfpython.c:213:76: error: passing argument 3 of ‘PyBytes_AsStringAndSize’ from incompatible pointer type [-Wincompatible-pointer-types]
  213 |         if( PyBytes_Check( pRc ) && ( PyBytes_AsStringAndSize( pRc, &cstr, &len ) != -1 ) ) {
      |                                                                            ^~~~
      |                                                                            |
      |                                                                            int *
In file included from /usr/include/python3.12/Python.h:50,
                 from tfpython.h:6,
                 from tfpython.c:10:
/usr/include/python3.12/bytesobject.h:57:17: note: expected ‘Py_ssize_t *’ {aka ‘long int *’} but argument is of type ‘int *’
   57 |     Py_ssize_t *len     /* pointer to length variable or NULL */


Changing int to long int on line 204 compiles without issue.

Like I said in the first one... I really don't know C at all, and was just trying to find something that would compile so that someone who knows better than me might be able to save a slight amount of time. :)